parser/src/parser/file.rs

```diff
@@ -359,20 +375,45 @@ impl ParserContext {
         Ok(members)
     }
 
+    fn parse_member_variable_name_and_type(&mut self) -> Result<(Identifier, Type)> {
```

Comment:
```suggestion
    fn parse_typed_variable(&mut self) -> Result<(Identifier, Type)> {
```

Reply:
I avoided that name if only because I didn't want to create confusion on why this isn't used for variable parsing outside of circuits as those take an optional type later on. I can still change it if you think it's better.

Reply:
Maybe `parse_typed_field_name` instead?